nuxeo-core/nuxeo-core-test/src/test/java/org/nuxeo/ecm/core/TestSQLRepositoryAPI.java

```diff
@@ -1227,6 +1227,31 @@ public void testGetParentDocument() {
         assertEquals(root.getPathAsString(), shouldBeRoot.getPathAsString());
     }
 
+    @Test
+    public void testGetParentDocumentFromPathReference() {
+        DocumentModel root = session.getRootDocument();
+        DocumentModel shouldBeNull = session.getParentDocument(new PathRef(root.getPathAsString()));
```

Comment:
We usually rely directly on `"/"` instead of `session.getRootDocument().getPathAsString()` to make it simpler, but it's a detail.

Reply:
I tried to be consistent with other tests in this Classe but I will simplify it

Reply:
There's a lot of old cruft in these tests, it's not always wise to reuse the style of the existing code 😀